package.json

```diff
@@ -3,6 +3,10 @@
   "version": "1.0.0",
   "description": "The ultimate Turbo / Stimulus / Hotwire modal window for Rails",
   "main": "dist/modal_controller.min.js",
+  "module": "dist/modal_controller.min.js",
+  "files": [
```

Comment:
If we don't include the `"files"` key here, I'm pretty sure npm won't serve your `/dist` folder.